torch_xla/csrc/lowering_context.cpp

```diff
@@ -45,7 +47,7 @@ class HloMetadataSetter {
     std::string op_type =
         absl::StrReplaceAll(node->op().ToString(), {{":", "_"}});
     metadata.set_op_type(op_type);
-    const ir::MetaData& nmeta = node->metadata();
+    const torch::lazy::MetaData& nmeta = node->metadata();
```

Comment:
This looks like a bug. You kept the xla version of the `MetaData` in Node class but in here `node->metadata()` will return the `torch::lazy::MetaData`. I am surprised this code actually compiles.